uspekhi-gerontologii.csl

```diff
@@ -0,0 +1,173 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" class="in-text" default-locale="ru">
+  <info>
+    <title>Uspekhi-gerontologii(numeric, sorted alphabetically, Russian)</title>
+    <id>http://www.zotero.org/styles/uspekhi-gerontologii</id>
+    <link href="http://www.zotero.org/styles/uspekhi-gerontologii" rel="self"/>
+    <link href="http://www.gersociety.ru/information/uspexi/" rel="documentation"/>
+    <author>
+      <name>Valery Mukhin</name>
+      <email>Valery.Mukhin@gmail.com</email>
+    </author>
+    <category citation-format="numeric"/>
+    <category field="generic-base"/>
```

Comment:
Please change `<category field="generic-base"/>` to `<category field="medicine"/>`.